pavelib/prereqs.py

```diff
@@ -31,7 +31,11 @@
 
 # Developers can have private requirements, for local copies of github repos,
 # or favorite debugging tools, etc.
-PRIVATE_REQS = 'requirements/philu/base.txt'
+if 'TOXENV' in os.environ:
+    PRIVATE_REQS = 'requirements/philu/testing.txt'
+else:
+    PRIVATE_REQS = 'requirements/philu/base.txt'
+
```

Comment:
Can we use ternary operator here?

`PRIVATE_REQS = 'requirements/philu/testing.txt' if 'TOXENV' in os.environ else 'requirements/philu/base.txt'`